burnman/mademember.py

```diff
@@ -0,0 +1,290 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+from __future__ import absolute_import
+
+import numpy as np
+import warnings
+
+from .mineral import Mineral, material_property
+from .solidsolution import SolidSolution
+from .solutionmodel import *
+from .averaging_schemes import reuss_average_function
+from .processchemistry import sum_formulae
+from . import constants
+
+
+class MadeMember(Mineral):
+
+    """
+    This is the base class for all solid solutions.
```

Comment:
do you now derive ``SolidSolution`` from ``MadeMember``? If yes, what is the difference between the two?